SAMtags.tex

```diff
@@ -253,6 +253,29 @@ \subsection{Metadata}
 \end{description}
 
 \subsection{Barcodes}
+DNA barcodes can be used to identify the provenance of the underlying reads.
+There are currently three varieties of barcodes that may co-exist: Sample Barcode, Cell Barcode, and Unique Molecular Identifier (UMI).
+\begin{itemize}
+\item
+Despite its name, the {\it Sample} Barcode identifies the {\it Library} and allows multiple libraries to be combined and sequenced together.
+After sequencing, the reads can be separated according to this barcode and placed in different ``read groups'' each corresponding to a library.
+Since the library was generated from a sample, knowing the library should inform of the sample.
+The barcode itself can be included in the {\tt PU} field in the {\tt RG} header line.
+Since the {\tt PU} field should be globally unique, it is advisable to include specific information such as flowcell barcode and lane.
+The {\tt ID} field of the {\tt RG} header line is not an appropriate place for the barcode as it is treated as mutable by several tools as it might be modified, for example when merging files.
```

Comment:
This sentence says the same thing twice. It would be clearer as e.g.

> It is not recommended to use the barcode as the {\tt ID} field of the {\tt RG} header line, as some tools modify this field (e.g., when merging files).